lib/checks/aria/is-element-focusable-evaluate.js

```diff
@@ -0,0 +1,7 @@
+import { isFocusable } from '../../commons/dom';
+
+function isElementFocusableEvaluate(node) {
+	return isFocusable(node);
```

Comment:
Could you pass the `virtualNode` here instead of the `node`? We've been trying to use `node` less.

```suggestion
function isElementFocusableEvaluate(node, options, virtualNode) {
	return isFocusable(virtualNode);
```